### PR TITLE
ui: update job description capitalization

### PR DESCRIPTION
**Pull request by kev-cao**
Changed job description capitalization to only first letter.

Epic: none

Release note: none

### CODE REVIEW

**Comment by kev-cao** [comment]
Screenshot of update:

image

File: pkg/ccl/backupccl/restore_online.go [hunk 1]
```diff
@@ -430,7 +430,7 @@ func (r *restoreResumer) maybeCalculateTotalDownloadSpans(
 
 	if err := r.job.NoTxn().Update(ctx, func(txn isql.Txn, md jobs.JobMetadata, ju *jobs.JobUpdater) error {
 		md.Progress.GetRestore().TotalDownloadRequired = total
-		md.Progress.RunningStatus = fmt.Sprintf("Downloading %s of restored data...", sz(total))
+		md.Progress.RunningStatus = fmt.Sprintf("Downloading %s of backup data...", sz(total))
```
Comment (msbutler) [inline pkg/ccl/backupccl/restore_online.go:433]: well, the data has already been restored via the linking phase! i don't think we should change this?

Comment (kev-cao) [reply pkg/ccl/backupccl/restore_online.go:433]: Ah that's what the rationale for it was. @alicia-l2 what are your thoughts on this?

Comment (alicia-l2) [reply pkg/ccl/backupccl/restore_online.go:433]: Oh this makes sense oops! Thanks for clarifying

Comment (kev-cao) [reply pkg/ccl/backupccl/restore_online.go]: Reverted this back to the original wording

**Review by xinhaoz — APPROVED** [review]

**Comment by kev-cao** [comment]
TFTR!

bors r=xinhaoz

**Pull request merged**